v1/model.md

```diff
@@ -355,6 +355,7 @@ A DocumentVersion is identified by the following attributes:
 | checksumAlgorithm    |      | string             | Checksum algorithm of the document version                          |
 | fileSize             |      | integer            | File size of the document version                                   |
 | contentType          |      | string             | Content type of the document version                                |
+| text                 |      | string             | Extracted text from the document version, if any                    |
```

Comment:
Not strictly related to this PR, but I got remember now when seeing the last part of the `.text` field description..

One thing that I've wonder quite a few times lately when consuming data from different endpoints, is which fields (if any) are nullable.

Is `.. if any` in the description here a hint it can be null, or more just an empty `string`?

Reply:
My intention was to imply that it could be null, yes, but I agree that it'd be better if we are explicit about such.

I'll make a note for myself to add this information when I get the time.